HandsOnTutorial/1-hello-world.fsx

```diff
@@ -17,47 +21,43 @@ common with MBrace.
 
 A guide to creating a cluster is [here](http://www.m-brace.net/#try).
 
-> NOTE: Before running, build this solution to get the required nuget packages, and edit credentials.fsx to enter your Azure connection strings.
+> NOTE: Before running, build this solution to get the required nuget packages, and edit Azure.fsx to enter your Azure connection strings.
 
 First connect to the cluster using a configuration to bind to your storage and service bus on Azure.
-*)
-let cluster = MBraceAzure.GetHandle(config)
-
-// Optionally, attach console logger to client object 
-cluster.EnableClientConsoleLogger <- true
+*)    
 
 // You can connect to the cluster and get details of the workers in the pool:
 cluster.ShowWorkerInfo()
 
-// You can view the history of processes:
-cluster.ShowProcessInfo()
-
 (** Now execute your first cloud workflow and get a handle to the running job: *)
-let job = 
+let task = 
     cloud { return "Hello world!" } 
-    |> cluster.CreateProcess
+    |> cluster.CreateCloudTask
 
-// You can get details for the process.
-job.ShowInfo()
+// You can get details for the task.
+task.ShowInfo()
 
 // Block until the result is computed by the cluster
-let text = job.Result
+let text = task.Result
 
 (** Alternatively we can do this all in one line: *)
 let quickText = 
     cloud { return "Hello world!" } 
-    |> cluster.Run
+    |> cluster.RunOnCloud
```

Comment:
Why not ``Run``.  A cloud task gets run in the cluster - that's not surprising.  RunOnCloud just poses more questions which a beginner shouldn't need to ask ("is the cloud different to the cluster", "why can I run anywhere else but the cloud"...)